src/styles/quiz-card/quiz-card.css

```
@@ -0,0 +1,59 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
```

The `gap` property for flexbox is not supported in all browsers yet. I would change this to a grid.
```suggestion
  display: grid;
  gap: 4px;
```